microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -84,6 +84,7 @@ For more information, see:
 * [Interop of native-side and web-side code](../how-to/communicate-btwn-web-native.md)
 * [Call web-side code from native-side code](../how-to/javascript.md)
 * [Call native-side code from web-side code](../how-to/hostobject.md)
+* [WebView2Script Package](/microsoft-edge/webview2/reference/index.yml) for `window.chrome.webview` JavaScript APIs
```

Comment:
```suggestion
* [WebView2Script Package](/microsoft-edge/webview2/reference/index.yml) for `window.chrome.webview` JavaScript APIs
* [Call native-side WinRT code from web-side code](../how-to/winrt-from-js.md)
```
"main" branch has the 2nd line, adjacent to 1st line, so considers it a "conflict" or "possible conflict".  Adding the 2nd line, from "main" branch, should fix.  

1 other file has a conflict as well.